src/v/cluster/controller_backend.cc

```diff
@@ -373,6 +373,10 @@ ss::future<> controller_backend::reconcile_ntp(deltas_t& deltas) {
     bool stop = false;
     auto it = deltas.begin();
     while (!(stop || it == deltas.end())) {
+        if (has_non_replicable_op_type(*it)) {
+            ++it;
+            continue;
+        }
```

Comment:
1. this definitely needs a comment because it looks related to correctness but in fact appears to be related to reducing logging.
2. is this important? early on in development perhaps the extra logging will be useful for debugging?

Reply:
For point 2 the logging can get crazy, for every no-op on every shard the `controller_backend` is emitting 2 additional logs. This makes it hard to parse to see the interesting logs from `reconciliation_backend`